.github/workflows/ci.yml

```diff
@@ -17,17 +17,26 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Set up Go
-        uses: actions/setup-go@v3
+        id: setup-go
+        uses: actions/setup-go@v4
         with:
           go-version-file: go.mod
-          cache: true
+          check-latest: true
 
       - name: Set up Chrome
         uses: browser-actions/setup-chrome@latest
 
       - name: Run lint
         uses: reviewdog/action-golangci-lint@v2
 
+      - name: Run govulncheck
+        uses: golang/govulncheck-action@v1
+        with:
+           go-version-input: '${{ steps.setup-go.outputs.go-version }}'
```

Comment:
GREAT! 👍

Reply:
 fm-
https://github.com/k1LoW/runn/actions/runs/5567078381/jobs/10168693130#step:7:98

Will it be okay?
https://github.com/k1LoW/runn/actions/runs/5567078381/jobs/10168693130#step:7:32353

Reply:
`Cannot open: File exists`

Perhaps it's because the previous step, actions/setup-go, has already created the file.